types/xk6-sql-browser/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// Type definitions for non-npm package xk6-sql 0.1
+// Project: https://github.com/grafana/xk6-sql
+// Definitions by: Leandro Gomes <https://github.com/leandrodotec>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.8
```

Comment:
```suggestion
```
that version is no longer enforced or specifically tested here, just remove this line.